packages/cli/src/commands/test.js

```diff
@@ -120,7 +122,10 @@ export const handler = async ({
 
   jestArgs.push(
     '--config',
-    `"${require.resolve('@redwoodjs/testing/config/jest/jest.config.js')}"`
+    `"${path.resolve(
```

Comment:
Was this change necessary @callingmedic911 ? I imagine require.resolve with the new file name would work right?

Reply:
You're right, I was trying `preset` here which required the directory path instead of the file, so had to use. Cleaned this up.